code/game/verbs/who.dm

```diff
@@ -1,63 +1,98 @@
-
 /client/verb/who()
 	set name = "Who"
 	set category = "OOC"
 
-	var/msg = "<b>Current Players:</b>\n"
+	var/list/lines = list()
+	var/list/temp = list()
+
+	for(var/client/C in GLOB.clients)
```

Comment:
Presumably `GLOB.clients` would only contain clients?
```suggestion
	for(var/I in GLOB.clients)
		var/client/C = I
```

Reply:
For a list with a max of 150 things that isn't being iterated on a loop, the performance gain here will be negligible.

Also sometime GLOB.clients can end up with nulls inside so this accounts for that.